packages/core/src/internal/proc.js

```diff
@@ -420,7 +420,7 @@ export default function proc(env, iterator, parentContext, parentEffectId, meta,
   }
 
   function resolveIterator(iterator, effectId, meta, cb) {
-    proc(env, iterator, taskContext, effectId, meta, cb)
+    proc(env, iterator, taskContext, effectId, meta, false, cb)
```

Comment:
imho already hard to read. could we extract in a separate variable?
```js
const isRoot = false
proc(env, iterator, taskContext, effectId, meta, isRoot, cb)
```

Reply:
okay, I'll add an inline comment here.

```javascript
proc(env, iterator, taskContext, effectId, meta, /* isRoot */ false, cb)
```